ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -45,7 +46,7 @@ def test_paddle_abs(
     dtype_and_x,
     on_device,
     fn_tree,
-    frontend,
+    frontend: Literal["paddle"],
```

Comment:
and also please revert all changes here and keep this file unchanged, thanks!

Reply:
thanks on it.